cloudformation.yaml

```diff
@@ -79,7 +81,7 @@ Resources:
         S3Bucket: sponsorship-expiry-email-lambda-dist
         S3Key: !Sub flexible/${Stage}/sponsorship-expiry-email-lambda/sponsorship-expiry-email-lambda.zip
       Handler : com.gu.comdev.sponsorshipexpiry.Lambda::handleRequest
-      MemorySize : 128
+      MemorySize : 512
```

Comment:
Has the Lambda failed at all and therefore needed more memory?

Reply:
Yes. The lambda was already set to 512 in the console and when I did the stack update for #3 I did a test run and it failed with an out-of-memory.